optimade/filtertransformers/mongo.py

```diff
@@ -275,9 +338,7 @@ def apply_length_op(subdict, prop, expr):
             return subdict
 
         return recursive_postprocessing(
-            filter_,
-            check_for_length_op_filter,
-            apply_length_op,
+            filter_, check_for_length_op_filter, apply_length_op
```

Comment:
Are these whitespace changes intentional? Would prefer to keep the diffs small

```suggestion
            filter_,
            check_for_length_op_filter,
            apply_length_op,
```